doc/source/release-notes/Percona-Server-5.6.27-75.0.rst

```diff
@@ -0,0 +1,67 @@
+.. rn:: 5.6.27-75.0
+
+==============================
+ |Percona Server| 5.6.27-75.0 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.27-75.0 on November 5th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.27-75.0/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.27 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-27.html>`_, including all the bug fixes in it, |Percona Server| 5.6.27-75.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.27-75.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.27-75.0>`_.
+
+New Features
+============
+
+ |Percona Server| is now available for *Ubuntu* 15.10 (Wily).
+
+ |TokuDB| MTR tests have been integrated into |Percona Server|.
+ 
+ Linux thread ID is now available in the :table:`PROCESSLIST` table. 
+
+ |Percona Server| has now re-enabled savepoints in triggers and stored functions.
+
+ Variables :variable:`innodb_buffer_pool_populate` and :variable:`numa_interleave` have been mapped to the upstream implementation of the new `innodb_numa_interleave <http://dev.mysql.com/doc/refman/5.6/en/innodb-parameters.html#sysvar_innodb_numa_interleave>`_ option.
+
+Bugs Fixed
+==========
+
+ Fixed transactional inconsistency with rollback in |TokuDB|  where large inserts within a transaction ended up with some rows committed and some rolled back after executing a rollback. Bug fixed :bug:`1512455`.
+
+ Variable :variable:`tokudb_backup_exclude` was not excluding files correctly. Bug fixed :bug:`1512457`.
+
+ |TokuDB| could crash if run in transaction-isolation was set to ``READ-COMMITTED``. Bug fixed :bug:`1501633`.
```

Comment:
"|TokuDB| could crash under load if transaction-isolation level ``READ-COMMITTED`` is used. Bug fixed :bug:`1501633`."